internal/backend/server.go

```diff
@@ -207,7 +207,7 @@ func (s *Server) Run(ctx context.Context) error {
 
 	servicesServer := grpcomni.MakeServiceServers(
 		s.state.Default(),
-		s.omniRuntime.CachedState(),
+		s.omniRuntime,
```

Comment:
is this intentional?

Reply:
yeah, I pass the runtime as the whole to the resource server now. It needs to get the dependency graph from it